client-lib/src/main/java/com/ibm/ws/repository/resources/internal/RepositoryResourceImpl.java

```diff
@@ -576,8 +581,7 @@ public AttachmentResourceWritable addContent(File file) throws RepositoryExcepti
     public AttachmentResourceWritable addContent(File file, String name) throws RepositoryException {
         if (_contentAttached) {
             throw new RepositoryResourceValidationException("addContent(" + file.getAbsolutePath()
-                                                            + ") called for resource " + getName() + " which all ready has a CONTENT attachment",
```

Comment:
Not sure what happened in this file. Can we get rid of all the spurious changes so that we're just left with the change you wanted to make?